release/air_tests/air_benchmarks/workloads/pytorch_training_e2e.py

```diff
@@ -92,6 +92,9 @@ def main(data_size_gb: int, num_epochs=2, num_workers=1):
     )
     print(f"Training for {num_epochs} epochs with {num_workers} workers.")
     start = time.time()
+    # Enable cross host NCCL for larger scale tests
```

Comment:
Should we update the nightly test to run with the silver configuration?

Reply:
hmm ideally yes but i wasn't sure what our config yaml should be for VPC / secure group to ensure cross host NCCL. My workspaces had a PlacementGroup with secure group setup by default that took care of it.

Reply:
hmm is specifying the runtime env not sufficient?

Reply:
you're right i only had it in my workspaces, looking for full YAML config for nightly now ..